xds/src/test/java/io/grpc/xds/ControlPlaneRule.java

```diff
@@ -86,9 +87,15 @@ public class ControlPlaneRule extends TestWatcher {
   private XdsTestControlPlaneService controlPlaneService;
   private XdsTestLoadReportingService loadReportingService;
   private XdsNameResolverProvider nameResolverProvider;
+  private final int port;
 
   public ControlPlaneRule() {
+    this(0);
+  }
+
+  public ControlPlaneRule(int port) {
```

Comment:
This file seems not related to the change. It is used for fallback?

Reply:
Yes, reverted it